pull_request_template.md

```diff
@@ -0,0 +1,44 @@
+<!-- 
+Hello Axolotl!
+Thank you for contributing to the lakeFS project.
+We appreciate the time invested in this pull request and created this template to help make this process easier.
+It's really important to have all the information and context, to ensure we can properly address this PR 
+Please use the following references to fill out the pull request.
+--> 
+
+### Linked Issue
+
+Closes #(Insert issue number closed by this PR)
+
+---
+
+## Change Description
+
+### Bug Fix
+
```

Comment:
I'd like to suggest the following sub-sections for bug fixes: 
* problem - the reason for opening the bug
* root cause - after investigating the bug we know the root cause 
* solution 

WDYT?